core/scss/components/select/select.scss

```diff
@@ -26,9 +26,17 @@
       width: 100%;
       @include button-color(
         $color: $select__color,
+        $color-css-var: --md-button-secondary-text-color,
+        $color-hover: $select__color,
+        $color-hover-css-var: --md-button-secondary-text-color,
+        $color-active: $select__color,
+        $color-active-css-var: --md-button-secondary-text-color,
```

Comment:
Looks like some of these didn't exist in the original, assuming they're necessary.

Reply:
I added them since it should act like a button. Not having a hover feels wrong. I'm pretty sure the design specs have a hover state as well, but can't find it.